### PR TITLE
add bluesky

### LINKED ISSUE #846: Add blusky social media link
- This is our profile: https://bsky.app/profile/rOpenSci.hachyderm.io.ap.brid.gy

### PR DESCRIPTION
**Pull request by maelle**
Fix #846

- Add link in the footer.
- Add link on author profile pages if their (Markdown) data includes Bluesky.
- Add link on team member info if their (JSON) data includes Bluesky.
- Add "share on Bluesky" links near blog posts/events/etc. (where we have the other share buttons... I simply looked for all occurrences of LinkedIn in non-content files).

@yabellini I couldn't find your profile link so @noamross is the demo!

### CODE REVIEW

**Comment by maelle** [comment]
for my future reference if needed https://discourse.gohugo.io/t/is-there-a-url-encoder-function-in-hugo/2239/8

File: themes/ropensci/layouts/partials/skeleton/head.html [hunk 1]
```diff
@@ -15,11 +15,7 @@
     {{ $css := resources.Get "main.min.css" }}
 {{ $secureCSS := $css | resources.Fingerprint "sha512" }}
   <link href="{{ $secureCSS.Permalink }}" rel="stylesheet" integrity="{{ $secureCSS.Data.Integrity }}">
- <link
-      rel="stylesheet"
-      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css
-"
-    />
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.6.0/css/all.min.css" integrity="sha512-Kc323vGBEqzTmouAECnVceyQqyqdsSiqLQISBL29aUW4U/M7pSPA/gEUZQqv1cwx4OnYxTxve5UMg5GT6L4JJg==" crossorigin="anonymous" referrerpolicy="no-referrer" />
```
Comment (maelle) [inline themes/ropensci/layouts/partials/skeleton/head.html:18]: I had to update our Font Awesome library to get the Bluesky icon cc @jeroen

**Comment by yabellini** [comment]
> @yabellini could you please check that the share on bluesky link works? for instance from [deploy-preview-847--ropensci.netlify.app/blog/2024/09/12/labels-writing-projects](https://deploy-preview-847--ropensci.netlify.app/blog/2024/09/12/labels-writing-projects/)
> 
> The docs I found are [docs.bsky.app/docs/advanced-guides/intent-links](https://docs.bsky.app/docs/advanced-guides/intent-links). What I am unsure about is whether I escaped the text parameter correctly.

This is working; it opens my blue sky and creates a post for me to share.

**Comment by maelle** [comment]
ok so fine to merge I suppose unless @noamross you don't want your bluesky profile listed on the website.

File: themes/ropensci/layouts/partials/skeleton/head.html (hunk 1 above)
Comment (jeroen) [reply themes/ropensci/layouts/partials/skeleton/head.html:18]: This does not break naming of our existing icons, right?

Comment (maelle) [reply themes/ropensci/layouts/partials/skeleton/head.html:18]: no they look good

**Comment by noamross** [comment]
Looks good to me, thanks!

**Pull request merged**